postgres/problems/problem_12.md

```
@@ -1,23 +1,21 @@
 ## 説明
-各注文 (`order_items`) について、
-商品ごとの数量が **注文全体の何 % を占めるか** を求め、
-割合 (`quantity_ratio_percent`) を小数点 2 桁で表示してください。
+`products` テーブルの平均価格を求め、その平均価格を上回る商品の id、name、price を取得してください。
 
 ## 制約
-* `SUM(quantity) OVER (PARTITION BY order_id)` を用いて分母を計算すること
-* パーセント計算は 100 x (個数 / 合計個数)
-* 出力は必ず **`ORDER BY order_id, quantity_ratio_percent DESC`** を付け、
-  まず注文 ID (`order_id`) 昇順、その中で割合が **大きい順（降順）** に並べること
+* WHERE 内でサブクエリを使用すること
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)